docs/editor/accessibility.md

```diff
@@ -158,7 +158,9 @@ Certain features such as folding and minimap (code overview) are disabled when i
 
 ## Terminal accessibility
 
-Output in the Integrated Terminal can be navigated through by using the "navigation mode" commands available in the Command Palette (press `kbstyle(F1)` and search for "terminal navigation mode").
+Terminal accessibility help can be toggled via `kbstyle(Alt+F1)` which describes some useful tips when using a screen reader. One of the tips described is to use `kbstyle(Shift+Tab)` to access the terminal's buffer, this will enter the screen reader's browse mode automatically (depending on your screen reader) and provide an accessible view of the entire terminal's buffer.
```

Comment:
maybe this is a bug, but terminal accessibility help cannot be toggled with that keybinding at the moment - only turned on.

Reply:
I don't think that's a bug as the editor does the same and we say how it can be hidden (focus elsewhere or via escape)